AudioKit/Common/Nodes/Playback/Player/AKAudioPlayer.swift

```diff
@@ -236,6 +236,7 @@ open class AKAudioPlayer: AKNode, AKToggleable {
     }
     
     deinit {
+        AKLog("* AKAudioPlayer")
```

Comment:
Is this descriptive enough?  I for one wouldn't know what this meant when it appeared in the console.